### DISCUSSION TITLE
How to get custom resolution?

### DISCUSSION THREAD

**Discussion by JerzyKruszewski** [hlky/stable-diffusion-webui#931] [Q&A]
Is there a way to change the width and height increment from 64 to 1?
Right now I can only choose 64=>128=>192=>...=>1024 but I need to make 480x270 (with x4 superresolution will give me FullHD)
I can change the initial value in the webui.yaml config, but it is unsustainable "fix"

EDIT: I actually tried to change it by modifying the config, as stated above. But this resulted in some kind of exception. So I wouldn't recommend it.

**Comment by outlookhazy** [comment]
I haven't tested, but you could probably modify frontend/frontend.py lines 30 and 32 to change the step from 64 to something smaller
